cea/demand/thermal_loads.py

```diff
@@ -101,14 +104,16 @@ def calc_thermal_loads(building_name, bpr, weather_data, date_range, locator,
             tsd['mcpcdata_sys'] = tsd['Tcdata_sys_re'] = tsd['Tcdata_sys_sup'] = np.zeros(HOURS_IN_YEAR)
             tsd['Edata'] = tsd['E_cdata'] = np.zeros(HOURS_IN_YEAR)
 
-        #CALCULATE HEATING AND COOLING DEMAND
+        #CALCULATE SPACE CONDITIONING DEMANDS
         tsd = calc_set_points(bpr, date_range, tsd, building_name, config, locator)  # calculate the setpoints for every hour
         tsd = calc_Qhs_Qcs(bpr, tsd, use_dynamic_infiltration_calculation)  #end-use demand latent and sensible + ventilation
         tsd = sensible_loads.calc_Qhs_Qcs_loss(bpr, tsd) # losses
         tsd = sensible_loads.calc_Qhs_sys_Qcs_sys(tsd) # system (incl. losses)
         tsd = sensible_loads.calc_temperatures_emission_systems(bpr, tsd) # calculate temperatures
         tsd = electrical_loads.calc_Eauxf_ve(tsd) #calc auxiliary loads ventilation
         tsd = electrical_loads.calc_Eaux_Qhs_Qcs(tsd, bpr) #calc auxiliary loads heating and cooling
+        tsd = calc_Qcs_sys(bpr, tsd) # final : including fuels and renewables
+        tsd = calc_Qhs_sys(bpr, tsd) # final : including fuels and renewables
```

Comment:
please take a second look at this. this seems a double calculation.... there is this function calc_Qhs_sys_Qcs_sys.
for renewables, they are accounted different, that is inside E_hs_PV etc...